app/views/publishers/vacancies/job_applications/_messages.html.slim

```diff
@@ -4,30 +4,22 @@
       = govuk_button_link_to t(".print_page"), "#", class: "govuk-button--secondary", "data-controller": "utils", "data-action": "click->utils#print" unless params[:show_form] == "true"
       = govuk_button_link_to t(".send_message_to_candidate"), url_for(params.to_unsafe_h.merge(show_form: "true")), class: "govuk-button--secondary govuk-!-margin-left-3" unless params[:show_form] == "true"
 
-    - if params[:show_form] == "true"
-      div class="govuk-!-margin-top-6"
-        = govuk_details summary_text: t(".how_will_message_be_sent") do
-          = t(".message_sent_description")
+    = render "shared/message_form",
+             message_object: @message_form,
+             form_url: organisation_job_job_application_messages_path(vacancy.id, job_application.id),
+             form_help_text: t(".how_will_message_be_sent"),
+             form_help_content: t(".message_sent_description"),
+             current_user_type: "publisher"
 
-        = form_for message, url: organisation_job_job_application_messages_path(vacancy.id, job_application.id), method: :post do |f|
-          = f.govuk_error_summary
-
-          .govuk-form-group
-            = f.rich_text_area :content
-
-          = f.govuk_submit t(".send_message")
-          = govuk_link_to t(".cancel"), request.original_url, class: "govuk-!-margin-left-3"
-    - else
+    - unless params[:show_form] == "true"
+      / Inset text (only show when form is not visible)
```

Comment:
This seems to undo a deliberate 'only check show_form once' idiom, so not sure why this has changed?